.gitignore

```diff
@@ -5,6 +5,7 @@
 # python
 *.egg-info/
 *.py[cod]
+pdm.lock
```

Comment:
Since this project uses `uv` now, this file shouldn't be created anymore, and you can just delete it if you still have one.

```suggestion
```